lib/mysql.js

```diff
@@ -332,9 +332,6 @@ MySQL.prototype.getInsertedId = function(model, info) {
  */
 MySQL.prototype.toColumnValue = function(prop, val) {
   if (val == null) {
-    if (prop.autoIncrement || prop.id) {
-      return new ParameterizedSQL('DEFAULT');
```

Comment:
I'm not certain what the best practice here would be; do we want to have some sort of `DEFAULT` setting, or give people the option to set a default value if they configure it to work this way?